tools/zbfakebeacon

```diff
@@ -49,7 +49,7 @@ if __name__ == '__main__':
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-f', '--channel', '-c', action='store', dest='channel', required=True, type=int, default=11)
-    parser.add_argument('-i', '--interface', action='store', dest='devstring')
+    parser.add_argument('-i', '--interface', action='store', dest='devstring', required=True)
```

Comment:
We have never had -i defined in tools as most interfaces are auto-discoverable if one is not provided, and thus users may not specify one if they are doing something simple.

Reply:
The -i parameter was already included.
I added the "required" argument, to get a correct error message when not using an interface. Otherwise the program quits with the error "KillerBee doesn´t understand device given by 'None'". Auto-discover does not work for me.